font-types/src/name_id.rs

```diff
@@ -153,6 +153,10 @@ impl NameId {
     /// If present in a variable font, it may be used as the family prefix in the PostScript Name Generation
     /// for Variation Fonts algorithm.
     pub const VARIATIONS_POSTSCRIPT_NAME_PREFIX: Self = Self(25);
+
+    /// Invalid `name_id`.
+    /// Value to represent a nonexistent name ID.
+    pub const INVALID: Self = Self(0xFFFF);
```

Comment:
I think we should prefer `Option` rather than in-band signaling whenever possible. I have a suggestion for this below.